sky/execution.py

```diff
@@ -677,6 +677,7 @@ def spot_launch(
             # Note: actual spot cluster name will be <task.name>-<spot job ID>
             'dag_name': dag.name,
             'retry_until_up': retry_until_up,
+            'skypilot_runtime_env': constants.SKY_REMOTE_PYTHON_ENV,
```

Comment:
Should we put this variable into `controller_utils.shared_controller_vars_to_fill` since it is both used by serve & spot controller?